config/application.rb

```diff
@@ -15,6 +15,8 @@ class Application < Rails::Application
     config.cache_max_age = ENV['CACHE_MAX_AGE'] || 10.seconds
 
     config.middleware.use Rack::BounceFavicon
+    require 'bounce_browserconfig'
+    config.middleware.use BounceBrowserconfig
```

Comment:
I had considered it but decided against it purely as a gut reaction to this being a part of our own codebase. Can be changed if you feel strongly enough about it.